src/views/preview/project-view.jsx

```diff
@@ -109,6 +110,7 @@ class Preview extends React.Component {
             socialOpen: false,
             favoriteCount: 0,
             isProjectLoaded: false,
+            isProjectCommentsGloballyEnabled: false,
```

Comment:
i think you want to remove this from the state object, right?

Reply:
Yes! Thank you.